grafana-plugin/src/style/vars.css

```diff
@@ -65,6 +65,7 @@
   --tag-text-success: rgb(50, 96, 43);
   --tag-border-success: rgb(148, 203, 140);
   --button-background: rgba(36, 41, 46, 0.16);
+  --box-background: rgba(244, 245, 245);
```

Comment:
We'll need to figure out at some point these variables, for now I couldn't find an existent variable that matches for both light and dark theme